src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -113,7 +113,7 @@ private void SetUsesNullableAttributes()
         /// This method handles duplicate types in a few different ways:
         /// - for types before C# 7, the first candidate is returned with a warning
         /// - for types after C# 7, the type is considered missing
```

Comment:
nit: the comment mentions "before C# 7" and "after C# 7". It's not clear which of these meant to be including C# 7 itself.